.changeset/seven-shrimps-build.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rdfjs/types": minor
```

Comment:
Not sure whether this should be minor or patch. It's a significant change but maybe still just a bug fix?

Reply:
I'm ok with classifying it as minor